website/src/pages/BlogPage/index.tsx

```diff
@@ -118,15 +119,15 @@ const BlogPage: React.FC = () => {
         break;
     }
   };
-
+  
   return (
     <>
     {currentBlogDetails?.id && (
       <Metadata 
         title={currentBlogDetails?.title} 
         description={currentBlogDetails?.excerpt} 
         url={currentLocation} 
-        image={`${currentOrigin}/images${currentPathname}.png` || `${currentOrigin}/images/blog/defaultImage.png`} 
+        image={`${currentOrigin}/images${currentPathname}.png`} 
```

Comment:
We won't require the OR condition anymore??

Reply:
yes